cdap-app-templates/cdap-etl/cdap-data-pipeline3_2.12/pom.xml

```diff
@@ -108,11 +108,17 @@
     <dependency>
       <groupId>io.cdap.common</groupId>
       <artifactId>common-http</artifactId>
+      <version>${cdap.common.version}</version>
     </dependency>
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
```

Comment:
Please use commons-lang3 instead. It also has a dependency management section in cdap pom.xml, so you don't need to specify verison

Reply:
Should I add it directly as a dependency? There seems to be some [conflict](https://cdap.atlassian.net/browse/CDAP-12742) already, and commons-lang3 is marked as excluded in both of this pom files.

Reply:
Hm. Great question. Probably let's skip it altogether and just do it as plain old non-null + empty check

Reply:
Done